src/Navigation.js

```diff
@@ -11,19 +10,23 @@ class Navigation extends Component {
   render() {
 
     return (
-      <Navbar bg="light" expand="lg">
-        <Navbar.Brand>
-          <Link to='/' >Rcruitme</Link>
-        </Navbar.Brand>
-        <Nav className="mr-auto">
-        </Nav>
 
+      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+        <Navbar.Brand href="/">
```

Comment:
Similar question re: full reload vs. client-side navigation. Is this for styling reasons?